components/DBTableMocker.php

```diff
@@ -10,11 +10,18 @@
     class DBTableMocker {
 
         /**
-         * Name of the column which is being changed
+         * Contains an array of stored table columns
          *
-         * @var string
+         * @var string[][]
          */
-        private $currentColumn = "";
+        private $columns = [];
```

Comment:
Columns have to be stored so that the 'column' parameter can go through the validation.